MCDataProducts/inc/ProcessCode.hh

```diff
@@ -85,10 +85,10 @@ namespace mu2e {
       truncated,       mu2eMuonCaptureAtRest,  mu2eMuonDecayAtRest,       mu2eCeMinusEndpoint, // 167
       mu2eCeMinusLeadingLog,   mu2eCePlusEndpoint,  mu2eDIOLeadingLog, mu2eInternalRMC,  // 171
       mu2eExternalRMC,         mu2eFlateMinus,      mu2eFlatePlus, mu2eFlatPhoton, // 175
-      mu2eCePlusLeadingLog, mu2ePionCaptureAtRest, mu2eExternalRPC, mu2eInternalRPC,
-      mu2eCaloCalib, mu2eunused6, mu2eunused7, mu2eunused8,
-      uninitialized, NoProcess, GammaGeneralProc,
-      mu2eGammaConversion, Radioactivation,
+      mu2eCePlusLeadingLog, mu2ePionCaptureAtRest, mu2eExternalRPC, mu2eInternalRPC, // 179
+      mu2eCaloCalib, mu2eunused6, mu2eunused7, mu2eunused8, // 183
+      uninitialized, NoProcess, GammaGeneralProc, // 187
+      mu2eGammaConversion, Radioactivation, nCaptureHP, nFissionHP, // 191
```

Comment:
I think that the last two numbers  should be 186 and 190.

Reply:
fixed